conda/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+    name: cosima-cookbook
+    version: {{ GIT_DESCRIBE_TAG}}
+
+source:
+    git_rev: master
+    git_url: ../
+
+build:
+    noarch: python
+    number: {{ GIT_DESCRIBE_NUMBER }}
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python>=3.6
+        - setuptools
+        - pbr
+    run:
```

Comment:
Maybe it comes in through another dependency, but I'm surprised not to see sqlalchemy here?

Reply:
Me too. Good catch. I just copied the `meta.yaml` I'd used to build this manually, and it wasn't in there. I'll add.